sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessor.java

```diff
@@ -64,28 +50,22 @@
      * @param eventHubAsyncClient The {@link EventHubAsyncClient}.
      * @param consumerGroupName The consumer group name used in this event processor to consumer events.
      * @param partitionProcessorFactory The factory to create new partition processor(s).
-     * @param initialEventPosition Initial event position to start consuming events.
-     * @param partitionManager The partition manager.
-     * @param eventHubName The Event Hub name.
+     * @param initialEventPosition The event position to start processing events from a partition if no checkpoint is
```

Comment:
Should we consider having this on a per-partition level?   Since checkpoints and other concepts apply to a given partition, it seems limiting to have this as a single unit applied to all.  My vote would be to remove this and offer guidance on how to bootstrap a checkpoint if there's a need to set a position other than "first."   If we're going to keep it as a parameter, I believe that we really need to make this a map of `partitionId => eventPosition`